### ISSUE TITLE
Starting in v3.23.4, descriptions no longer display for responses

### ISSUE THREAD

**Issue by scrohde** [swagger-api/swagger-ui#5604]
### Q&A (please complete the following information)
 - OS: any
 - Browser: any
 - Version: v3.23.4
 - Method of installation: dist assets
 - Swagger-UI version: v3.23.4
 - Swagger/OpenAPI version: OpenAPI 3.0

### Content & configuration

Example Swagger/OpenAPI definition:
```yaml
  /nfv/cntlr:
    get:
      tags:
      - users
      summary: Controller Version
      operationId: cntlrShow
      responses:
        "200":
          description: success
          content:
            application/json:
              schema:
                $ref: schemas/users.json#/definitions/cntlr
        default:
          description: |-
            Error Responses

            Expected HTTP status codes for this operation:
            - 400 - Bad Request
            - 401 - Unauthorized
            - 403 - Forbidden
            - 500 - Internal Server Error
            - 503 - Service Unavailable

            The response body will contain more information
          content:
            application/json:
              schema:
                $ref: schemas/common.json#/definitions/GigaErrors
```

Swagger-UI configuration options:
```js
SwaggerUI({
  // your config options here
})
```

```
?yourQueryStringConfig
```

### Describe the bug you're encountering
Descriptions in Response Object no longer displays in swagger-ui

### To reproduce...

Add a description to a OpenAPI 3.0.0 Response Object
Try to view it in swagger-ui v3.23.4 or greater

### Expected behavior
Should see the description

### Screenshots
In v3.23.3 it works just fine
[Selection_045.pdf](https://github.com/swagger-api/swagger-ui/files/3603115/Selection_045.pdf)

In v3.23.4 it no longer works
[Selection_046.pdf](https://github.com/swagger-api/swagger-ui/files/3603119/Selection_046.pdf)

**Comment by shockey** [comment]
@scrohde & @Alex314Ru, I'm not able to reproduce this using either Swagger UI v3.23.4's static assets or the latest Swagger UI.

Here's what I'm seeing, based on @scrohde's fragment of an OpenAPI document:

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/680248/64926628-8d868800-d7b4-11e9-9497-e2fc7cadb6e8.png">

Happy to look into this further, but there's not much I can do without being able see what's going on locally. More information, context, or a repository that reproduces the problem would be helpful!

**Comment by Alex314Ru** [comment]
@shockey That's strange. Today I installed v3.23.10, and now everything works fine for me.

**Comment by scrohde** [comment]
Yep, fixed in v3.23.10.

**Comment by webron** [comment]
Thanks for verifying!
